raaghu-layouts/src/rds-comp-app-shell/rds-comp-app-shell.tsx

```
@@ -1,20 +1,22 @@
-import React, { ReactNode, useState } from "react";
+import React, { ReactNode } from "react";
 import "./rds-comp-app-shell.css";
-import { BrowserRouter, Outlet } from "react-router-dom";
+import { Outlet } from "react-router-dom";
 import RdsCompTopNavigation from "../../../raaghu-components/src/rds-comp-top-navigation";
```

ESLint

[error] 'RdsCompTopNavigation' is defined but never used.

(@typescript-eslint/no-unused-vars)